bpython/cli.py

```diff
@@ -974,7 +974,7 @@ def p_key(self, key):
             try:
                 source = self.get_source_of_current_name()
             except repl.SourceNotFound as e:
-                self.statusbar.message(_(e))
+                self.status_bar.message('%s' % (e, ))
```

Comment:
That's what I had before, but then found the code in `bpython/curtsiesfrontend/repl.py` doing this.
Let's change both to use `str` then?